src/cli.rs

```diff
@@ -243,6 +243,20 @@ pub struct MqttConnection {
     )]
     pub client_key: Option<std::path::PathBuf>,
 
+    /// Path to the TLS CA certificate file.
+    ///
+    /// Used together with --client-cert, --client-key to enable TLS client
+    /// authentication.
+    /// The file has to be a DER-encoded X.509 certificate serialized to PEM.
+    #[arg(
+        long,
+        env = "MQTTUI_CA_CERTIFICATE",
+        value_hint = ValueHint::FilePath,
+        value_name = "FILEPATH",
+        global = true,
+    )]
+    pub ca_cert: Option<std::path::PathBuf>,
```

Comment:
Not sure about the name here. CA Certificate Path. value name already suggests path so not including that is a good idea. CA ist certificate authority. Sometimes this variable is called ca_cert and sometimes ca_certificate_path. Certificate authority certificate. Maybe authority_certificate is the most readable one?

Also I think, having the same variable name in all places is probably a good idea.